demo-app/app/components/accordion/index.hbs

```diff
@@ -31,8 +31,8 @@
   local-class="Accordion {{if this.isTriggerFocused "focus"}}"
   {{on "focusin" this.handleFocusin}}
   {{on "focusout" this.handleFocusout}}
-  {{on-key "ArrowUp" this.jumpToPreviousTrigger event="keyup"}}
-  {{on-key "ArrowDown" this.jumpToNextTrigger event="keyup"}}
+  {{on-key-mod "ArrowUp" this.jumpToPreviousTrigger event="keyup"}}
+  {{on-key-mod "ArrowDown" this.jumpToNextTrigger event="keyup"}}
```

Comment:
Just a double check, but is this change correct? Does this modifier exist with this name?

Reply:
Ah. yes, this warrants an explanation. The way glint deals with non-strict templates is to let you add types in a global registry. There is a single registry that includes components and helpers and modifiers. ember-keyboard has both a helper named `on-key` and a modifier named `on-key`. This is impossible to work with as-is with the system I just described. As a workaround, in this PR, we re-export the `on-key-mod` modifier as `on-key-mod` and add that to the registry.

Reply:
Ahhh! That explains indeed.